--- downloads/Healpix_3.20/hpxconfig_functions.sh	2015-06-22 21:21:21.503470248 -0700
+++ downloads/Healpix_3.20/hpxconfig_functions_patched.sh	2015-06-22 21:21:25.200955182 -0700
@@ -85,7 +85,7 @@
 }
 #-------------
 findFITSLib () {
-    for dir in $* /usr/lib /usr/lib64 /usr/local/lib /usr/local/lib64 /usr/local/lib/cfitsio /usr/local/lib64/cftisio /usr/local/src/cfitsio ${HOME}/lib ${HOME}/lib64 ./src/cxx/${HEALPIX_TARGET}/lib/ /softs/cfitsio/3.24/lib /usr/common/usg/cfitsio/3.26/lib ; do
+    for dir in $* /usr/lib /usr/lib64 /usr/local/lib /usr/local/lib64 /usr/local/lib/cfitsio /usr/local/lib64/cftisio /usr/local/src/cfitsio ${HOME}/lib ${HOME}/lib64 ./src/cxx/${HEALPIX_TARGET}/lib/ /softs/cfitsio/3.24/lib /usr/common/usg/cfitsio/3.26/lib /usr/lib/i386-linux-gnu /usr/lib/x86_64-linux-gnu ; do
 	if [ -r "${dir}/lib${LIBFITS}.a" -o -r "${dir}/lib${LIBFITS}.so" -o -r "${dir}/lib${LIBFITS}.dylib" ] ; then
 	    FITSDIR=$dir
 	    break
