--- downloads/Healpix_3.20/src/C/subs/Makefile	2015-06-22 21:25:10.144663170 -0700
+++ downloads/Healpix_3.20/src/C/subs/Makefile_patched	2015-06-22 21:24:35.913805309 -0700
@@ -107,7 +107,7 @@
 shared: libchealpix$(SHLIB_SUFFIX) #tests
 
 libchealpix$(SHLIB_SUFFIX) : $(OBJD)
-	$(SHLIB_LD) -o $@ $(OBJD)
+	$(SHLIB_LD) -o $@ $(OBJD) $(CFITSIO_LIBS)
 
 # Make the dynamic (Mac) library itself
 dynamic: libchealpix$(DYLIB_SUFFIX) #tests
